scripts/flashida/write_method.py

```
@@ -0,0 +1,105 @@
+    'MS2',
+    {'key': 'Analyzer', 'name': 'Analyzer', 'value': 'Orbitrap'},
+    {'key': 'FirstMass', 'name': 'FirstMass', 'value': 200},
+    {'key': 'OrbitrapResolution', 'name': 'OrbitrapResolution', 'value': 60000},
+    {'key': 'AGCTarget', 'name': 'AGCTarget', 'value': 500000},
+    {'key': 'MaxIT', 'name': 'MaxIT', 'value': 118},
+    {'key': 'Microscans', 'name': 'Microscans', 'value': 1},
+    {'key': 'DataType', 'name': 'DataType', 'value': 'Centroid'},
+    {'key': 'Activation', 'name': 'Activation', 'value': 'HCD'},
+    {'key': 'CollisionEnergy', 'name': 'CollisionEnergy', 'value': 29},
+    'IDA',
+    {'key': 'MaxMs2CountPerMs1', 'name': 'MaxMs2CountPerMs1','value': 4},
```

Flake8 (7.2.0)

[error] missing whitespace after ','

(E231)